spec/features/publish/accepting_terms_spec.rb

```diff
@@ -20,6 +21,10 @@
     then_i_should_see_an_error_message
   end
 
+  def given_the_new_publish_flow_feature_flag_is_enabled
+    allow(Settings.features).to receive(:new_publish_navigation).and_return(true)
```

Comment:
I think we could make use of the `enable_features` helper. Eg:

```ruby
before do
  enable_features(: new_publish_navigation)
end

after do
  disable_features(:new_publish_navigation)
end
```

Reply:
Good shout too - have done